core/orchestrator_core.go

```diff
@@ -4294,6 +5542,18 @@ func (o *TridentOrchestrator) DeleteNode(ctx context.Context, nodeName string) (
 	return nil
 }
 
+// deleteNode deletes a node from the backend.
```

Comment:
This is a bit misleading.  As with the previous function, this one deletes a node from Trident, and it removes access to that node from all backends.  We may have to push a commit atop the AI generated content to fix those things it doesn't quite grok.